config/fact_check.yml

```diff
@@ -4,6 +4,7 @@
 # If we ever want to change this format, we'll need to support some notion of
 # legacy formats, so we still pick up emails to old addresses
 address_format: <%=ENV.fetch("FACT_CHECK_ADDRESS_FORMAT", "factcheck+dev-{id}@alphagov.co.uk") %>
+subject_format: <%=ENV.fetch("FACT_CHECK_SUBJECT_FORMAT", "Fact Checked [dev-{id}]") %>
```

Comment:
This needs to make sense to the recipient of the original fact check email. I suggest it's as close as possible to the current subject, but including the ID. 

Current format is `‘[content title]’ GOV.UK preview of new edition` 
eg ‘[Car and trailer driving test]’ GOV.UK preview of new edition

Reply:
That's a good point — currently, the [code](https://github.com/alphagov/publisher/pull/1256/files#diff-b5c2ae4ebc91d294985cc4742f1889abR20) assumes that the ID is the only changeable part of the input but this won't be the case. I'll have to think about how to implement that.